README.md

```diff
@@ -203,6 +203,16 @@ npm run test
       - Locate the **terra-framework_devcontainer** under "Dev Containers"
       - Hover over the dev container and click the Folder icon labelled **Open Folder in Container** or by right clicking and selecting **Open Folder in Container**
 
+<h3>
+  Refreshing the container
+</h3>
+
+If any of the compose files were to be changed, you must refresh the docker container by running the following command:
```

Comment:
Can we add this: `delete the corresponding existing docker images` and then run the command. 
Also, having this under step 6 would make sense since this is only related to wdio and devcontainer docker image can be rebuilt using vscode.